src/styles/main.scss

```diff
@@ -1,7 +1,23 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "./utils/mixins.scss";
```

Comment:
Please import the 'fonts' file to include the Sharp Sans No2 Medium and Avenir fonts as mentioned in the task description.